hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/commit/files/PendingSet.java

```diff
@@ -47,8 +47,8 @@
  */
 @InterfaceAudience.Private
 @InterfaceStability.Unstable
-public class Pendingset extends PersistentCommitData {
```

Comment:
Well it's two words. Pending and Set. So it's camel cased. Maybe `PendingBatch`?

The issue about case changing and macos doesn't make sense. The patch wasn't merged yet so no one will have issues with the name change.